Library/Formula/duplicity.rb

```diff
@@ -1,8 +1,7 @@
 class Duplicity < Formula
   homepage "http://www.nongnu.org/duplicity/"
-  url "http://code.launchpad.net/duplicity/0.6-series/0.6.25/+download/duplicity-0.6.25.tar.gz"
-  sha1 "fe0b6b0b0dc7dbc02598d96567954b48c4308420"
-  revision 1
+  url "http://code.launchpad.net/duplicity/0.7-series/0.7.02/+download/duplicity-0.7.02.tar.gz"
```

Comment:
You can bump this to ` https:// `. There's a redirect from 80 to 443, but 80 should be avoided altogether if 443 is available.